src/distributing-policies/secure-supply-chain.md

```diff
@@ -0,0 +1,567 @@
+
+```console
+$ kwctl scaffold verification-config > verification_config.yaml
+$ cat verification_config.yaml
+# Default Kubewarden verification config
+#
+# With this config, the only valid policies are those signed by Kubewarden
+# infrastructure.
+#
+# This config can be saved to its default location (for this OS) with:
+#   kwctl scaffold verification-config > /home/jvanz/.config/kubewarden/verification-config.yml
+#
+# Providing a config in the default location enables Sigstore verification.
+# See https://docs.kubewarden.io for more Sigstore verification options.
+---
+apiVersion: v1
+allOf:
+  - kind: githubAction
+    owner: kubewarden
+    repo: ~
+    annotations: ~
+anyOf: ~
+
+$ kubectl  create configmap my-signatures-configuration --from-file=verification-config=verification_config.yaml
+configmap/my-signatures-configuration created
+$ kubectl  get configmap -o yaml my-signatures-configuration
+apiVersion: v1
+data:
+  verification-config: |+
+    # Default Kubewarden verification config
+    #
+    # With this config, the only valid policies are those signed by Kubewarden
+    # infrastructure.
+    #
+    # This config can be saved to its default location (for this OS) with:
+    #   kwctl scaffold verification-config > /home/jvanz/.config/kubewarden/verification-config.yml
+    #
+    # Providing a config in the default location enables Sigstore verification.
+    # See https://docs.kubewarden.io for more Sigstore verification options.
+    ---
+    apiVersion: v1
+    allOf:
+      - kind: githubAction
+        owner: kubewarden
+        repo: ~
+        annotations: ~
+    anyOf: ~
+
+kind: ConfigMap
+metadata:
+  creationTimestamp: "2022-04-07T11:54:27Z"
+  name: my-signatures-configuration
+  namespace: default
+  resourceVersion: "1317"
+  uid: 74dec846-7fcd-4b4b-8184-700c816f685a
+
+```
+
+After creating the `configmap` to store the signature requirements, the users
+can configure a Policy Server to start validating policy signatures by setting the
+`configmap` name in the field `verificationConfig`.
+
+```yaml
+apiVersion: policies.kubewarden.io/v1alpha2
+kind: PolicyServer
+metadata:
+  name: default
+  finalizers:
+    - kubewarden
+spec:
+  image: ghcr.io/kubewarden/policy-server:v0.2.7
+  serviceAccountName: policy-server
+  replicas: 1
+  verificationConfig: your_configmap   #name of the confimap with the signatures requirements
+  env:
+    - name: KUBEWARDEN_ENABLE_METRICS
+      value: "1"
+    - name: KUBEWARDEN_LOG_FMT
+      value: otlp
+    - name: "KUBEWARDEN_LOG_LEVEL"
+      value: "info"
+
+```
+
+If you deploy the default Policy Server using the `kubewarden-defaults`
+Helm chart, you can configure this field by setting the `configmap` name in the
+`policyServer.verificationConfig` value.
+
+Now, the PolicyServer will reject untrusted AdmissionPolicies and ClusterAdmissionPolicies,
+and those would never be in active status or able to process admission review requests.
```

Comment:
This would be nice, but that's not actually the case :(

It would be cool to have our kubewarden-controller's validating admission webhook reject the creation of `AdmissionPolicy` and `ClusterAdmissionPolicies` that do not use trusted policies. Maybe we should create an issue for that. Could you do that please? :pleading_face: 

However, currently the behaviour is going to be different: Policy Server will refuse to start if even one of the policies referenced by the user cannot be trusted.

Reply:
Done!